src/Containers/Profile/Profile.scss

```diff
@@ -0,0 +1,194 @@
+@import '../../assets/sass/global.scss';
+#profile {
+  input,
+  button {
+    width: 80%;
+  }
+  button {
+    width: 80%;
+  }
+  label {
+    color: #444;
+  }
+  textarea {
+    border: 2px solid #efefef;
+    margin-bottom: 10px;
+    padding: 0 6px;
+    width: 90%;
+    background: #fafafa;
+    &:focus {
+      outline: none !important;
+      border: 2px solid rgba(#aca9a9, 0.5);
+      box-shadow: 2px 0 6px rgba(#efefef, 0.8);
+    }
+  }
+  .navbar {
+    background-color: white;
+  }
+}
+
+.body {
+  padding-left: 14%;
+
+  div,
+  .profile {
+    background-color: #f5f5f5;
+  }
+  #pic {
+    img {
+      width: 70%;
+    }
+    .uname {
+      padding: 50px 10;
+    }
+  }
+}
+@media only screen and (max-width: 700px) {
+  #profile {
+    padding-left: 10%;
+    #pic {
+      display: flex;
+      height: 100px;
+      .form-box {
+        width: 40%;
+        height: 0;
+        padding-top: 0%;
+        img {
+          width: 110%;
+        }
+      }
+
+      .uname {
+        padding: 0;
+      }
+    }
+  }
+  input,
+  .button {
+    width: 90% !important;
+  }
+  textarea {
+    width: 90%;
+    margin: 5px 0;
+    padding: 3px;
+  }
+}
+.p-avatar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 120px;
+  height: 120px;
+  border-radius: 50%;
+
+  .avatar-main {
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.12);
+    background-size: cover;
+    margin-bottom: 20%;
+  }
+  .edit {
+    background-color: #f46036;
+    border-radius: 50%;
+    position: absolute;
+    height: 28px;
+    width: 28px;
+    margin-left: 2.8rem;
+    margin-top: 1.6rem;
+    text-align: center;
+    padding: 4px;
+    color: white;
+    font-size: 14px;
+    cursor: pointer;
+  }
+}
+.col-sm-3 {
```

Comment:
I can see that you are editing grid class directly, can you try to give another class to the element and edit that class instead? the reason is that we are all using those classes for the layout and it can't be obvious to troubleshoot if we are all editing them in our styles.